pkg/ebpf/log.go

```diff
@@ -0,0 +1,55 @@
+package ebpf
+
+import (
+	"fmt"
+	"github.com/aquasecurity/tracee/pkg/metrics"
+	log "github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus/hooks/writer"
+	"io/ioutil"
+	"os"
+)
+
+// ErrorChanWriter is a io.Writer implementation that writes to error channel, while updating program stats.
+type ErrorChanWriter struct {
+	c     chan<- error
+	stats *metrics.Stats
+}
+
+func InitErrorChanWriter(c chan<- error, stats *metrics.Stats) ErrorChanWriter {
+	return ErrorChanWriter{
+		c:     c,
+		stats: stats,
+	}
+}
+
+func (ec ErrorChanWriter) Write(message []byte) (n int, err error) {
+	ec.stats.ErrorCount.Increment()
+	ec.c <- fmt.Errorf(string(message))
+	return n, nil
+}
+
+// ConfigureLogger configure the errors created by tracee to be directed into the error channel, and normal
+// logs to stderr.
+// Notice that the logger log level must enable the log for it to be written.
+// For example, if the followed log level is Error and above, Warnings won't be written to the errors channel.
```

Comment:
Just to make sure, does the ordering of the levels in `LogLevels` matter, or does it just declare the relevant levels and their order is static?

Reply:
No, the order is insignificant here.